plugins/de.fraunhofer.ipa.roscode.generator/src/de/fraunhofer/ipa/roscode/generator/RosCodeGenerator.xtend

```diff
@@ -141,36 +152,46 @@ void  «sub.name»_cb (const «sub.message.package.name»::«sub.message.name»
 
 int main(int argc, char **argv)
 {
-  ros::init(argc, argv, "«node.name»");
-  ros::NodeHandle n;
-  
-            «FOR pub : node.publisher»
-  «pub.compile»
-            «ENDFOR»
-            «FOR sub : node.subscriber»
-  «sub.compile»
-            «ENDFOR»
-            «FOR srvserver : node.serviceserver»
-  «srvserver.compile»
-            «ENDFOR»
-            «FOR srvclient : node.serviceclient»
-  «srvclient.compile»
-            «ENDFOR»
-
-  ros::spin();
-
-  return 0;
+	ros::init(argc, argv, "«node.name»");
+  	ros::NodeHandle n;
+  	«FOR param : node.parameter»
+  	«IF (param.type instanceof ParameterStructTypeImpl)»
+  	«parameter_helper.compile_struct(param.type as ParameterStructTypeImpl, param.name).getKey()»
+  	«ELSE»
+  	«var param_pair = parameter_helper.get_param_type(param.type)»
+  	«IF !(param_pair.getKey().empty)»
+  	«param_pair.getKey()» «param.name»_;
+  	n.param("«param.name»", «param.name»_«param_pair.getValue()»);
```

Comment:
a " is missed at the end.

It should be 

> n.param("«param.name»", «param.name»_«param_pair.getValue()»**"**);

Reply:
The problem is here actually: https://github.com/ipa320/ros-model/blob/5886dc0a821e65cb4a95d424898eb1bf602127d4/plugins/de.fraunhofer.ipa.roscode.generator/src/de/fraunhofer/ipa/roscode/generator/RosCodeGenerator.xtend#L54